content/en/mobile_testing/mobile_app_tests/results.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Mobile App Test Results
+kind: documentation
+description: View Synthetic mobile app test results and compare successful or failed sample runs to test runs. 
+further_reading:
+- link: "/mobile_testing/mobile_app_tests/"
+  tag: "Documentation"
+  text: "Learn about Synthetic mobile tests"
+- link: "/service_management/events/explorer"
+  tag: "Documentation"
+  text: "Learn about the Events Explorer"
+---
+
+## Overview
+
+Click on a mobile app test on the [**Synthetic Tests** page][11] to see the Test Details page. The Test Details page contains all the information relating to your test, including test properties, test history, sample runs, and test runs.
+
+{{< img src="mobile_testing/test_details.png" alt="The Test Details page" style="width=80%" >}}
+
+
+Test runs appear in a test details page after a Synthetic mobile app test executes. [Sample results](#sample-results) correlate to the latest passed and failed test executions over a time interval and in a specific number of locations and devices.
+
+## Test properties
+
+In the **Properties** section, you can see the test ID, test creation and edit dates, test priority, environment tag, and additional tags.
+
+**Overview** 
+: This section describes the test on which mobile application, version, location, number of devices, test interval, and the number of test steps.
```

Comment:
Can you rewrite this sentence? Grammatically it's a little award with the "the test on which mobile application" piece.

Reply:
```suggestion
: This section describes the Synthetic test details, including the mobile application, version, location, number of devices, test interval, and the number of test steps.
```